src/app/games/abalone/AbaloneTutorial.ts

```diff
@@ -50,8 +43,27 @@ export class AbaloneTutorial extends Tutorial {
                 [_, _, O, _, _, N, N, N, N],
             ], 0),
             [AbaloneMove.ofSingleCoord(new Coord(4, 4), HexaDirection.RIGHT)],
-            $localize`Congratulations! You know everything you need to start a game!`,
+            TutorialStepMessage.CONGRATULATIONS(),
             TutorialStepMessage.FAILED_TRY_AGAIN(),
         ),
+        TutorialStep.fromPredicate(
+            TutorialStepMessage.TRANSLATIONS(),
+            $localize`To translate a group of piece, you must select the first piece, then select the last piece, then click on one of the 4 lateral direction the group can move without touching opponent pieces.<br/><br/>Go ahead, translate some pieces.`,
```

Comment:
Meilleure formulation, avec moins de fautes, et d'après notre ami Chat:
"To translate a group of pieces, first select the first piece, then the last piece, and finally click on one of the four lateral directions in which the group can move without touching opponent pieces.<br/><br/>Go ahead, try translating some pieces."

Reply:
le "try" est vraiment nécessaire ?

Reply:
pas tant !